mParticle-iterable/MPKitIterable.m

```diff
@@ -32,6 +40,23 @@ - (void)start {
     
     dispatch_once(&kitPredicate, ^{
         
+        [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(onUserIdentified:) name:mParticleIdentityStateChangeListenerNotification object:nil];
```

Comment:
Can you switch to using the interfaces within kit protocol?

https://github.com/mParticle/mparticle-apple-sdk/blob/master/mParticle-Apple-SDK/Include/MPKitProtocol.h#L98-L101

The issue with the above is that if a customer filters customerid or email, the kit will still receive those, which is a privacy issue. I've flagged this as I noticed other kits doing the same.